src/Stoa.ts

```diff
@@ -438,18 +438,20 @@ class Stoa extends WebService
 
                 // TODO We should apply the block's timestamp to this method when it is added.
                 let out_put: Array<ITxHistoryElement> = [];
-                let base_time = Date.UTC(2020, 0);
                 for (const row of rows)
                 {
                     out_put.push({
+                        wallet_tx_type: row.wallet_tx_type,
```

Comment:
From Wallet Requirements Documentation
A visual representation (through a colored icon) of the type of transaction. If the transaction is inbound (funds are added), a green plus sign (“+”) is used. For outbound funds, a red minus sign (“-“) is used. For freezing transactions, a blue snowflake icon should be used. For data storage, the icon and color are not yet defined.